src/dra/73D7C.c

```diff
@@ -0,0 +1,162 @@
+#include "common.h"
+#include "dra.h"
+#include "objects.h"
+#include "sfx.h"
+
+s32 func_80113D7C(s16 arg0) {
```

Comment:
`func_80113D7C` and afterwards do not need to be in their own separate files. You can merge `73D7C.c` into `73AAC.c`